drape/drape_tests/CMakeLists.txt

```diff
@@ -31,7 +31,13 @@ target_compile_definitions(${PROJECT_NAME}
   PRIVATE GTEST_DONT_DEFINE_TEST WITH_GL_MOCK
 )
 
-add_subdirectory(../../3party/googletest ${CMAKE_CURRENT_BINARY_DIR}/googletest)
+SET(GOOGLE_TEST_MIN_VERSION "1.12.0")
+find_package(GTest ${GOOGLE_TEST_MIN_VERSION})
+if (${GTest_FOUND})
+  message(STATUS "The build will rely on the system provided GoogleTest version")
+else()
+  add_subdirectory(../../3party/googletest ${CMAKE_CURRENT_BINARY_DIR}/googletest)
```

Comment:
It would be great to update gtest with the latest version first manually, in a separate commit.

Reply:
All right, it will be here: https://github.com/organicmaps/organicmaps/pull/3538